### ISSUE TITLE
Columns 1px displacement

### ISSUE THREAD

**Issue by webdreamer** [DataTables/FixedColumns#67]
My fixed columns are displaced by 1 px, and I found the following code in the column calculation function:

```
border = $(that.s.dt.nTable).css('border-left-width');
iWidth += typeof border === 'string' ? 1 : parseInt( border, 10 );
```

The border-left-width is being read as "0 px" and, since that is a string, it's count as 1. Why? Why do you verify if typeof border is string, and do not trust parseInt when it is?

**Comment by DataTables** [comment]
The check was intended to be against things like "thin", "medium" etc. But looking at the code you pointed out I can see that is obviously BS and not working here.

Yup - that needs a fix - thanks for highlighting it. It will be fixed in the next release.

**Comment by DataTables** [comment]
It was fixed in 87036b591482983a118a4c7e1b253750298b55aa. Thanks again!
